.github/workflows/ci.yml

```diff
@@ -160,7 +161,7 @@ jobs:
           - "presto-sqlserver,presto-postgresql,presto-mysql"
           - "presto-oracle"
           - "presto-kudu"
-          - "presto-phoenix,presto-iceberg"
+          - "presto-phoenix,presto-iceberg,presto-druid"
```

Comment:
how much time does this group take now, and how much time druid adds?

Reply:
I am actually not sure about what rationale to use when deciding where should I add the `presto-druid` module? Is it duration a group takes that should determine this?

Reply:
Also, in this run https://github.com/prestosql/presto/pull/3522/checks?check_run_id=711960215, I don't think Druid integration tests ran. 
```
[INFO] presto-phoenix ..................................... SUCCESS [08:42 min]
[INFO] presto-iceberg ..................................... SUCCESS [06:35 min]
[INFO] presto-druid ....................................... SUCCESS [  5.386 s]
```
My guess is only the unit tests were executed. 

What needs to be done to run the integration test?

Reply:
I added https://github.com/prestosql/presto/pull/3522/commits/85fa53250c2b0b1a307a9fec5c4de2c51ce32bd5 
let me know if it helps.